com.unity.netcode.gameobjects/Components/Interpolator/BufferedLinearInterpolator.cs

```diff
@@ -4,13 +4,11 @@
 
 namespace Unity.Netcode
 {
-
     /// <summary>
     /// Solves for incoming values that are jittered
     /// Partially solves for message loss. Unclamped lerping helps hide this, but not completely
     /// </summary>
-    /// <typeparam name="T"></typeparam>
```

Comment:
Why remove this?

Reply:
doesn't have to be removed, it just didn't have a content